docs/source/get_involved/governance.rst

```diff
@@ -200,17 +217,23 @@ Becoming inactive in the above sense means not contributing for the period via:
 * commenting on pull requests or issues
 * attending one of the regular meetings
 
-Becoming active (after becoming inactive) in the above sense requires one of:
+of a scope that goes beyond algorithm maintainer expectations - i.e., PR or issues related to owned algorithms do not count.
+
+Becoming active (after becoming inactive) in the above sense requires:
 
-* an approved pull request authored by the core developer
-* a contribution to the community that is minuted in one of the regular meetings
+* a contribution within the last month that goes beyond algorithm maintainer expectations, e.g.:
+  * an approved pull request authored by the core developer
+  * a substantial contribution to the community that is minuted in one of the regular meetings
```

Comment:
```suggestion
```

Reply:
we can remove this bcz below you write there needs to be a vote as for newly appointed core-devs

Reply:
I think it should require both. Not just a vote, but actually having done something within the last month.